languages/fsharp/exercises/concept/values/.docs/introduction.md

```diff
@@ -0,0 +1,9 @@
+In F#, a _value_ is everything that has a type and can be defined. That includes booleans, integers and lists, but also functions. An integer value is defined as one or more (consecutive) digits and it supports the default mathetical operators.
+
+Associating a name with a value is referred to as a _binding_. A binding is defined using the `let` keyword. Function bindings are regular bindings, but with one or more parameters, which are separated by spaces. Invoking a function is done by specifying its name and passing arguments for each of the function's parameters, again separated by spaces.
+
+A key aspect of bindings is that they're immutable. This means that once a binding has been defined, its value can never change. As a consequence, a new binding has to be defined for changing values.
+
+Specifing the type of a binding is optional. If no type is specified for a binding, the F# compiler will use _type inference_ to try to automatically infer the type of the binding. It will do this based on the value of the binding. For functions, the type is inferred by analyzing what values the function is called with and what value the function returns.
```

Comment:
```suggestion
Instead of having "variables" or "constants", associating a name with a value is called a _binding_ in F#. A binding is defined using the `let` keyword. Bindings are immutable. This means that once a binding has been defined, its value can never change. As a consequence, a new binding must be defined for changing values.

Function bindings are the same as regular bindings, except that they have one or more parameters, which are separated by spaces. You can invoke a function by specifying its name and passing arguments for each of the function's parameters, again separated by spaces.

Specifying the type of a binding is optional. If no type is specified for a binding, the F# compiler will use _type inference_ to try to automatically infer the type of the binding. It will do this based on the value of the binding. For functions, the type is inferred by analyzing what values the function is called with and what value the function returns.
```

I've just reordered this slightly to make it flow a little more as I read it.